src/input.ts

```diff
@@ -23,12 +24,14 @@ export function get(): Input {
     }
     const useCross = input.getInputBool('use-cross');
     const name = input.getInput('name');
+    const argsFilePath = input.getInput('args-file')
 
     return {
-        token: input.getInput('token', {required: true}),
+        token: input.getInput('token', { required: true }),
```

Comment:
Might be good to avoid changing formatting to avoid pain in merges/rebases if we ever want to get upstream changes?

Reply:
Good idea. Changed.